README.md

```diff
@@ -60,29 +60,29 @@ This will work with `notepad++.exe`, however some functionality in Notepad++ wil
 
 To build SciLexer.dll without PCRE support (for both 32-bit & 64-bit):
 
- 1. For 32-bit, open a command prompt *for building* ([a.k.a. the *Developer Command Prompt for VS2013*](https://msdn.microsoft.com/en-us/library/f2ccy3wt.aspx))
-    - From the IDE, you can do this by right clicking on a file in Solution Explorer,
-      and clicking "Open Command Prompt". This will open up a command prompt with all the proper environment variables.
+ 1. For 32-bit Windows, open a command prompt *for building* ([a.k.a. the *Developer Command Prompt for VS2013*](https://msdn.microsoft.com/en-us/library/f2ccy3wt.aspx))
+    - From the IDE, you can do this by right-clicking on a file in Solution Explorer,
+      then clicking "Open Command Prompt" (this will open a command prompt with all the proper environment variables)
     - From the Windows Start screen/menu, type `Developer Command Prompt for VS2013`,
-      and click/select the result.
+      then click/select the result
     - From an *already open* command prompt, run `vcvarsall.bat`
-      (e.g. "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat").
+      (e.g., "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat")
 
-	For 64-bit, open *VS2013 x64 Native tool command prompt*.
+	For 64-bit Windows, open *VS2013 x64 Native tool command prompt*.

```

Comment:
This change introduces ambiguity. It's the SciLexer.dll that’s going to be 64-bits, (and therefore Notepad++), but not necessarily Windows.
I recommend changing this to "To build a 64-bits DLL, ..."